graphql/resolvers.go

```diff
@@ -1850,10 +1851,48 @@ func (r *queryResolver) TaskTestSample(ctx context.Context, tasks []string, test
 			}
 			testResultsToReturn = append(testResultsToReturn, tr)
 		}
-		return testResultsToReturn, nil
+	} else {
+		regexFilter := ""
+		for i, f := range testFilters {
+			regexFilter += f.TestName
+			// If there is another test filter add a pipe to separate them
+			if i < len(testFilters)-1 {
```

Comment:
I think a slightly cleaner way to do this is add it at the beginning of the loop if `i != 0` but both work

Reply:
I'm not sure if we can since we only want to add this condition if we know there is gonna be another filter proceeding the current one.

Reply:
It works since you add it to the top of the loop, since you immediately add the testname after:
```
for i, f := range testFilters {
    if i != 0 {  // add pipe }
    regexFilter := f.TestName
}
```